probe_scraper/parsers/histograms.py

```diff
@@ -85,7 +85,9 @@ def transform_probe_info(probes, version):
 class HistogramsParser:
     def parse(self, filenames, version=None, channel=None):
         # Call the histogram tools for each file.
-        parsed_probes = list(histogram_tools.from_files(filenames))
+        parsed_probes = list(
+            parse_histograms.from_files(filenames, strict_type_checks=False)
+        )
```

Comment:
vendored parser changed to require explicitly ignoring type checks. This fixes the previous warning: `Unable to parse whitelist (%s). Assuming all histograms are acceptable.`